rules/checked-by-xxx/rule.md

```diff
@@ -18,40 +18,50 @@ redirects:
 created: 2010-03-01T06:05:18.000Z
 archivedreason: null
 guid: 1aedc867-366f-47d3-aae8-9ff6d74da7ba
+
 ---
 
-There are times you should have your email checked by someone else before sending.
-Simply ask for that person to come over and check it on your computer or IM them if the person is not close by (e.g. in a different room).
-This way you are guaranteed instant feedback and you won't clog up someone else's inbox with unnecessary emails.
+There are times you should have your email checked by someone else before sending it. This way you are guaranteed instant feedback and you won't clog up someone else's inbox with unnecessary emails or mistakes.
+
+Simply ask for that person to come over and check it on your computer or call them and share your screen if the person is not nearby.
 
 <!--endintro-->
 
-Examples are:
+::: info
+**Warning:** Don't forget to Cc the person who checked your email.
+:::
+
+![Figure: Consider this the "four-eye principle" (also known as the "two-person rule" or "dual control")](four-eyes.jpg)
+
+Scenarios where having someone review your emails is recommended:
 
-* Where you are a little unsure
-* When you are new to a company and sending an email to a large distribution list
-* Sending a complicated email
-* Sending a sensitive email (where the content could be perceived as confrontational)
-* In application development, when you need a quick '[test please](/conduct-a-test-please)', and the person is immediately available. This provides you a chance to check and collaborate on the finished change before the '[done email](/dones-do-you-reply-done-and-delete-the-original-email)' is sent. It also shows the email was double-checked. 
+* You are new to a company
+* Sending an email to a large distribution list
+* You are a little unsure
+* You feel that it is a complicated email
+* Sending a sensitive email (e.g. The content could be perceived as confrontational)
+* You need a quick '[test please](/conduct-a-test-please)', and the person is immediately available. This provides you a chance to check and collaborate on the finished change before the '[done email](/dones-do-you-reply-done-and-delete-the-original-email)' is sent. It also shows the email was double-checked.
 
 Doing this will ensure:
 
 * You avoid grammatical and spelling errors
+* Your message is clear
 * The layout and tone of the email is professional
-* You haven’t forgotten anything important
+* You don’t forgotten anything important
```

Comment:
Change to 
    You don’t forget anything important

Reply:
done